src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -2237,9 +2239,17 @@ public static class ObservedData
       public static readonly string ObservedDataOrganDescription = "Organ where the data was measured";
       public static readonly string ObservedDataCompartmentDescription = "Compartment where the data was measured";
       public static readonly string ObservedDataMoleculeDescription = "Molecule for which the data was measured";
-      public static readonly string ORGAN = "Organ";
-      public static readonly string COMPARTMENT = "Compartment";
-      public static readonly string MOLECULE = "Molecule";
+      public static readonly string Organ = "Organ";
```

Comment:
@abdelr @georgeDaskalakis I think that constants that are USED as key should not be defined in UIConstants but in Constants. Then they can have different captions in Observed Date. But we do query after Organ and cOmpartment in PKSim explicitly and I don't like that we are breaking our convention

Reply:
moved them all (apart from the description ones) to Core.Constants nad renamed them so that they are uppercase.